api/all/src/main/java/io/opentelemetry/api/trace/ArrayBasedTraceStateBuilder.java

```diff
@@ -40,6 +40,16 @@ static TraceState empty() {
     this.parent = parent;
   }
 
+  /**
+   * Allows key value pairs to be added to the TraceState.
+   *
+   * @param key is an opaque string up to 256 characters printable. It MUST begin with a lowercase
+   *     letter, and can only contain lowercase letters a-z, digits 0-9, underscores _, dashes -,
+   *     asterisks *, and forward slashes /. For multi-tenant vendor scenarios, an at sign (@) can
+   *     be used to prefix the vendor name. The tenant id (before the '@') is limited to 240
+   *     characters and the vendor id is limited to 13 characters. If in the multi-tenant vendor
+   *     format, then the first character may additionally be numeric.
```

Comment:
Is this javadoc on the `TraceStateBuilder` interface already? If not, would it be better there?